_sass/custom/custom.scss

```diff
@@ -277,4 +277,7 @@ $text-color: #111111;
 	}
     }
 }
+.table-wrapper {
+  color: $black;
```

Comment:
I think green color will look more cool. what you think?

Reply:
Since the background is white i am not very sure about the green colour.